### DISCUSSION TITLE
Increasing Polyphony

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#259] [Ideas]
Not sure if doable, but could it be possible to increase the poly count of some parts, reducing the polyphony of another one if necessary ? or even unlock the poly cap altogether ?

**Comment by bangcorrupt** [comment]
The sequencer looks like it was designed for 8 note polyphony, then limited to 4 to match the available DSP power.  Most of the code reading notes from the pattern loops over the empty 4 bytes, so it might be possible to increase sequencer polyphony to 8 at some point.  This would be really useful for two handed piano parts, especially with external sound modules.
